2023/YoctoSummit-2023.11/fdo/fido-device-onboarding.md

```diff
@@ -0,0 +1,516 @@
+class: center, middle, intro
+
+# **FIDO Device Onboarding**
+
+# Late-binding Provisioning & Tales from the Trenches of Bleeding Edge Tech
+
+### Yocto Summit 2023
+
+## Tymoteusz Burak
+
+<img src="/remark-templates/3mdeb-presentation-template/images/logo.png"
+  width="150px" style="margin-left:-20px">
+
+---
+
+# `whoami`
+
+.center[<img
+  src="/remark-templates/3mdeb-presentation-template/images/tymek_burak.png"
```

Comment:
@tym2k1 that image does not render. Did you forget to push remark templates or add a submodule commit?

Reply:
Addresed in [remark-templates PR#8](https://github.com/3mdeb/remark-templates/pull/8)

Reply:
@tym2k1, it still doesn't work because you didn't update the revision to which submodule is checkout.